apps/labs/posts/building-scipy-with-flang.md

```diff
@@ -0,0 +1,684 @@
+---
+title: >
+  The 'eu' in eucatastrophe – Why SciPy builds for Python 3.12 on Windows are a minor miracle
+published: November 5, 2023
```

Comment:
We also need to fix the date; haven't committed yet, but once we're ready, feel free to edit here & commit.
```suggestion
published: November 8, 2023
```